packages/client/src/components/BigMovieCard/BigMovieCard.jsx

```diff
@@ -48,6 +57,12 @@ export const BigMovieCard = ({ currentMovie }) => {
     borderRadius: '1.25rem',
     margin: '2.5rem 0',
   });
+  // eslint-disable-next-line no-console
```

Comment:
Can you remove the comment and console.log, pls?

Reply:
🤦🏻‍♂️thanks for spotting this. Already removed it.